content/blog/explore-the-hpe-greenlake-for-compute-ops-management-rest-api-using-curl-and-postman.md

```diff
@@ -0,0 +1,333 @@
+
+* Sign up for an HPE Account 
+* Connect to HPE GreenLake
+* Create API Client Credentials 
+* Generate an Access Token based on your credentials and relevant connectivity endpoint (HPE GreenLake for Compute Ops Management in our case) 
+
+Once the access token is generated, it can be used to make API calls to perform any HTTP method against the desired HPE GreenLake for Compute Ops Management resource. To do this, you simply add the access token to the HTTP header with the keyword "Authorization: Bearer {token}". The name “Bearer authentication” can be understood as “give access to the bearer of this token”.
+
+There are several ways to invoke the API:
+
+* Using REST directly or from JavaScript 
+* Using cURL
+* Using Python
+* Using Go
+* Using PowerShell
+* Using Ansible, Terraform, etc.
+
+The [HPE GreenLake for Compute Ops Management API Reference site](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/openapi/compute-ops-latest/overview/) leverages an OpenAPI conformant documentation that provides a complete explanation of the operations supported by the Unique Resource Identifiers [(URIs)](<>)as well as sample requests and responses. 
+
+If you are wondering which to use, cURL is probably a good choice if you like command line interfaces, Postman if you prefer graphical user interfaces, and PowerShell or Python if you are really into programming.
+
+# Using cURL
+
+From the command prompt, use a simple cURL command like:
+
+```shell
+curl -X GET https://us-west2-api.compute.cloud.hpe.com/compute-ops/v1beta2/servers?limit=2 -H "Authorization:Bearer ,< access_token_here>”**
+```
+
+Note that you must use the correct connectivity endpoint according to the region where HPE GreenLake for Compute Ops Management is deployed. Currently, these are the connectivity endpoints for the possible HPE GreenLake for Compute Ops Management regions:
+
+* [EU Central](https://eu-central1-api.compute.cloud.hpe.com)   
+* [AP Northeast](https://eu-central1-api.compute.cloud.hpe.com) 
+* [US West](https://us-west2-api.compute.cloud.hpe.com)
+
+The request uses the **GET** method for the **/compute-ops/v1beta2/servers** resource to obtain the list of the available servers as described in the API reference.
+
+![blog figure3](/img/greenlake-com-ops-api-curl25.png)
+
+The command uses a header parameter with keyword **"Authorization:Bearer {token}**"
+
+**?limit=2** at the end of the URL is a query parameter to limit the response to a maximum of two servers as documented in the request pane of the *servers* resource:
+
+![blog figure4](/img/greenlake-com-ops-api-curl19.png)
+
+ As a result of the command, the API response lists two compute servers that are on boarded and assigned to the corresponding application for the user's account.
```

Comment:
I *think* there is a space before "As" that shouldn't be there. Take a look at the snip to see if you concur. Also, "on boarded" in this case should have a hyphen, so it should be "on-boarded".
![image](https://user-images.githubusercontent.com/77496261/205346164-2f1fc455-fc9e-43e6-947f-43b1ddc79c5f.png)